.git_archival.txt

```diff
@@ -0,0 +1,4 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true,match=*[0-9]*)$
+ref-names: $Format:%D$
```

Comment:
@Mr0grog FYI it was later determined that having `ref-names` is problematic, and it's no longer recommended to include it into this template: https://setuptools-scm.rtfd.io/en/latest/usage/#git-archives / https://github.com/pypa/setuptools_scm/pull/1033.

Reply:
Ah, that is interesting, thanks very much. I’ll pull it out of the main development branch. 🙇 

That said, please post things like this as a new issue (or an existing open issue if applicable) rather than re-opening discussion on a completed PR in the future. It makes it a little easier to track things that need to get done (and attribution for release notes). 🙂

Reply:
Yeah, it was more of a drive-by discovery, a thing that I noticed in passing so I put a comment where some context exist. I'm not a user of this project, but when I notice something in places I visit while doing git paleontology, and don't have time to spend on writing a long explanation, this seems like an acceptable middle ground :)